code/datums/dna.dm

```diff
@@ -436,6 +436,7 @@
 	for(var/datum/mutation/human/HM in mutations)
 		if(HM.type == A)
 			return HM
+	return FALSE
```

Comment:
Is this needed?

Returning null is the sane thing to do here, not FALSE.

Reply:
This is because if you're getting a mutation they don't have, then you should be returning "nothing"--aka NULL.

Reply:
Oh, oops. I'll remove it